src/common/src/Modal/ModalTopBar/ModalTopBar.test.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { render, fireEvent, queryByTestId, getByTestId } from '@testing-library/react';
+import ModalTopBar from './ModalTopBar';
+
+describe('<ModalTopBar />', () => {
+	it('should render', () => {
+		const { container } = render(<ModalTopBar>Foo</ModalTopBar>);
+		expect(container.firstElementChild).toBeDefined();
+		expect(getByTestId(container, 'modal-top-bar')).toBeDefined();
+		expect(getByTestId(container, 'modal-top-bar-children')).toBeDefined();
+		expect(getByTestId(container, 'modal-top-bar-divider')).toBeDefined();
+	});
+
+	it('should render with close button, when hideCloseButton is false', () => {
+		const { container } = render(<ModalTopBar hideCloseButton={false}>Foo</ModalTopBar>);
+		expect(getByTestId(container, 'modal-close-button')).toBeDefined();
+	});
+
+	it('should render without close button, when hideCloseButton is true', () => {
+		const { container } = render(<ModalTopBar hideCloseButton>Foo</ModalTopBar>);
+		expect(queryByTestId(container, 'modal-close-button')).not.toBeInTheDocument();
+	});
+
+	it('should render with close button as default', () => {
+		const { container } = render(<ModalTopBar>Foo</ModalTopBar>);
+		expect(getByTestId(container, 'modal-close-button')).toBeDefined();
+	});
+
+	it('Divider should not be transparent, when hideDivider is false', () => {
+		const { container } = render(<ModalTopBar hideDivider={false}>Foo</ModalTopBar>);
+		const divider = getByTestId(container, 'modal-top-bar-divider');
+		const style = window.getComputedStyle(divider);
+		expect(style.backgroundColor).not.toBe('transparent');
+	});
+
+	it('Divider should be transparent, when hideDivider is true', () => {
+		const { container } = render(<ModalTopBar hideDivider>Foo</ModalTopBar>);
+		const divider = getByTestId(container, 'modal-top-bar-divider');
+		const style = window.getComputedStyle(divider);
+		expect(style.backgroundColor).toBe('transparent');
+	});
+
+	it('Divider should not be transparent as default', () => {
+		const { container } = render(<ModalTopBar>Foo</ModalTopBar>);
+		const divider = getByTestId(container, 'modal-top-bar-divider');
+		const style = window.getComputedStyle(divider);
+		expect(style.backgroundColor).not.toBe('transparent');
+	});
+	/**
+	 *  Can't use getBoundClientRect because jsdom only returns the correct keys, not the actual values.
+	 *  See: https://github.com/jsdom/jsdom/issues/653
+	 */
+	it('should render with expected styles', () => {
+		const { container } = render(<ModalTopBar>Foo</ModalTopBar>);
+		// const header = getByTestId(container, 'modal-top-bar');
```

Comment:
Can be removed?

Reply:
I would like to keep this inline, because of the mentioned jsdom issue not being able to test on a actual size

Reply:
Ok, fair enough